From 00d1e2bcee72857c093a6588f7bceac3b8397809 Mon Sep 17 00:00:00 2001
From: Elad Alfassa <elad@fedoraproject.org>
Date: Thu, 21 Aug 2014 21:32:21 +0300
Subject: [PATCH] connection-editor: Don't show launcher in GNOME

As discussed with aday, control-center will spawn nm-connection-editor when
needed, and therefor we really don't need the launcher shown,
it clutters the Application list.

https://bugzilla.gnome.org/show_bug.cgi?id=682456
---
 nm-connection-editor.desktop.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/nm-connection-editor.desktop.in b/nm-connection-editor.desktop.in
index 55239d0..4eefa8a 100644
--- a/nm-connection-editor.desktop.in
+++ b/nm-connection-editor.desktop.in
@@ -10,4 +10,4 @@ X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=NetworkManager
 X-GNOME-Bugzilla-Component=nm-connection-editor
 Categories=GNOME;GTK;Settings;X-GNOME-NetworkSettings;
-NotShowIn=KDE;
+NotShowIn=KDE;GNOME;
-- 
2.1.0

